.github/workflows/php.yml

```diff
@@ -0,0 +1,23 @@
+name: PHP Composer
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Validate composer.json and composer.lock
+      run: composer validate
+
+    - name: Install dependencies
+      run: composer install --prefer-dist --no-progress --no-suggest
+
+    # Add a test script to composer.json, for instance: "test": "vendor/bin/phpunit"
```

Comment:
We could, but then it would raise the question of what we gain over travisCI. In travis, we currently launch parallel tests of phpunit for all supported php versions from 5.4 - 7.4. And we also run the mocha tests in all of these environments at the end.